src/class-wpml-cornerstone-register-strings.php

```diff
@@ -19,4 +19,12 @@ protected function register_strings_for_modules( array $data_array, array $packa
 	private function get_node_id( $data ) {
 		return md5( serialize( $data ) );
 	}
+
+	private function type_is_layout( $type ) {
+		return in_array(
+			str_replace( 'classic:', '', $type ),
```

Comment:
@dgwatkins where is `classic:` located in the `$type` string? At the beginning?

If so, perhaps it would be better to:
1. Define a constant in the class. E.g. `MODULE_TYPE_PREFIX = 'classic:';`
2. Use `substr( $type , strlen( self::MODULE_TYPE_PREFIX ) )`

It's probably unlikely that a module will contain 'classic:' somewhere else in the name, but it's better to not take any chance :)

Reply:
done :+1: